build/app.js

```diff
@@ -116,6 +117,11 @@ exports.execute = function(options) {
   var stageDir = utils.getFile(options.STAGE_DIR);
   utils.ensureFolderExists(stageDir);
 
+  var l10nTSPath = utils.joinPath(stageDir.path, 'timestamp-l10n.json');
+  if (utils.fileExists(l10nTSPath)) {
+    utils.deleteFile(l10nTSPath);
+  }
+
```

Comment:
Maybe I misunderstood here, for my understand is we want to read the previous timestamp-l10n.json to generate reproducible build. See also TimestampManager in multilocales.js, it tries to load timestamp-l10n.json, so I've no idea why you want to delete it here .